tests/unit/setup-rendering-test-test.js

```diff
@@ -39,7 +39,13 @@ describe('setupRenderingTest', function() {
       expect(this.element.textContent.trim()).to.equal('Pretty Color: green');
     });
 
-    it('renders a second time without', async function() {
+    it('renders with new color with same property', async function() {
+      this.set('name', 'red');
```

Comment:
@brunoocasali what happens if you change this line to `this.name = 'red'`? I'm investigating a ton of failures in our app since bumping to ember-source@3.13 and am curious if this is the same issue.

Reply:
At my company app using the way you told me worked for some of failing specs, but for some reason in others it doesn't. But I can't reproduce in the demo project.

For example https://gist.github.com/brunoocasali/a50170c59a35e75e7bcaaf078f4cff99
Breaks the last of specs...

Reply:
Awesome! :) 
Do you have any ideas about where is the problem origin? Like another lib for example?

Reply:
I did some initial bisecting and saw this ember-source PR: https://github.com/emberjs/ember.js/pull/18314 but am not familiar enough to say whether or not it is related. That said, downgrading ember-source does make the tests pass: https://github.com/emberjs/ember-mocha/compare/master...efx:failing-setter-test-downgrade-ember?expand=1
So it is something introduced in 3.13.x